lib/sms.php

```diff
@@ -59,26 +61,26 @@ function send_sms( $to, $message ) {
 function send_single_sms_via_rest( $body ) {
 	$endpoint = get_rest_url( 'Messages.json' );
 	$response = wp_remote_post( $endpoint, array(
-		'body' => $body,
+		'body'    => $body,
 		'headers' => array(
-			'Authorization' => 'Basic ' . base64_encode( TWILIO_SID . ':' . TWILIO_SECRET ), // {AccountSid}:{AuthToken}
-		)
+			'Authorization' => 'Basic ' . base64_encode( TWILIO_SID . ':' . TWILIO_SECRET ), // AccountSid:AuthToken
```

Comment:
Edited the comment because of "This comment is 72% valid code; is this commented out code?"